inc/deprecated/3.5.php

```diff
@@ -6,6 +6,11 @@
 /**
  * Class aliases.
  */
+class_alias( '\WP_Rocket\Admin\Settings\Page', '\WP_Rocket\Engine\Admin\Settings\Page' );
+class_alias( '\WP_Rocket\Admin\Settings\Render', '\WP_Rocket\Engine\Admin\Settings\Render' );
+class_alias( '\WP_Rocket\Admin\Settings\Settings', '\WP_Rocket\Engine\Admin\Settings\Settings' );
+class_alias( '\WP_Rocket\ServiceProvider\Settings', '\WP_Rocket\Engine\Admin\Settings\ServiceProvider' );
+class_alias( '\WP_Rocket\Subscriber\Admin\Settings\Page_Subscriber', '\WP_Rocket\Engine\Admin\Settings\Subscriber' );
 class_alias( '\\WP_Rocket\\Engine\\Preload\\AbstractPreload', '\\WP_Rocket\\Preload\\Abstract_Preload' );
```

Comment:
The class aliases that were already in this file are backwards:

```php
class_alias( 'the original class', 'the new class' );
```

[See the PHP manual](https://www.php.net/manual/en/function.class-alias.php). But let's fix these in a separate PR.